packages/@react-types/form/src/index.d.ts

```diff
@@ -30,5 +30,25 @@ export interface SpectrumFormProps extends DOMProps, AriaLabelingProps, StylePro
    * Whether the Form elements should display their "valid" or "invalid" visual styling.
    * @default 'valid'
    */
-  validationState?: ValidationState
+  validationState?: ValidationState,
+  /**
+   * Sets or retrieves the URL to which the form content is sent for processing.
+   */
+  action?: string,
+  /**
+   * Sets or retrieves the encoding type for the form.
+   */
+  encType?: 'application/x-www-form-urlencoded' | 'multipart/form-data' | 'text/plain' | string,
```

Comment:
Remove the string? Are there other options? Otherwise, we need to do the trick of `string & {}` to work around the TS bug like in `dna.d.ts`.

Reply:
I've revised the types after more research, explained in commit message